data/hexapod/gains18.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It looks like the XML and HRDF files are from Lily, not Daisy, right?  I think we should first get the Daisy ones into `master`, and then merge the other ones into the `Lily` branch later